python/sbp/client/drivers/pyserial_driver.py

```diff
@@ -75,13 +111,25 @@ def read(self, size):
         size : int
           Number of bytes to read.
         """
-        try:
-            return self.handle.read(size)
-        except (OSError, serial.SerialException):
-            print
-            print "Piksi disconnected"
-            print
-            raise IOError
+        # print('pre: consume acq')
```

Comment:
What happens when the serial FD goes away?  Does it throw the same exception as before?